cylc/flow/cfgspec/workflow.py

```diff
@@ -1334,6 +1337,28 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     "[platforms][<platform name>]submission retry delays"
                 )
             )
+            Conf(
+                'run mode', VDR.V_STRING,
+                options=list(TASK_CONFIG_RUN_MODES),
+                default=RunMode.LIVE.value,
+                desc=f'''
+                    When the workflow is running in live mode, run this *task*
+                    in one of the following modes:
+
+                    ``{RunMode.LIVE.value}`` (default):
+                        {RunMode.LIVE.describe()}
+                    ``{RunMode.SKIP.value}``:
+                        {RunMode.SKIP.describe()}
+
```

Comment:
@oliver-sanders and @wxtim What do you think about this?
```suggestion

                        .. note::

                           This is primarily intended for testing and
                           development; Cylc will warn you about any tasks
                           set to run in skip mode at validation time.
                           If you are using skip mode to create a dummy task,
                           you can ignore this warning.
```

Reply:
Just to outline some of the primary use cases for `run mode = skip`:

* [I want to skip a cycle of tasks](https://github.com/cylc/cylc-admin/blob/master/docs/proposal-interventions.md#3-i-want-to-skip-a-cycle-of-tasks-and-allow-the-workflow-to-continue) (e.g. my workflow has fallen behind and needs to skip cycles in order to catch up).
* I want to turn off a family/chain of tasks (e.g. task chain not needed for a given cycle for some external reason).
* I have a task that exists only for graphing reasons (e.g. workflow portability, family-family triggers, etc).

Reply:
These cases are documented in my docs PR, although the middle one could be expanded.

Reply:
```suggestion
                        .. note::

                           This is primarily intended to be set at runtime via
                           a broadcast; Cylc will warn you about any tasks
                           set to run in skip mode in the workflow
                           configuration at validation time.
```

Reply:
> * I have a task that exists only for graphing reasons (e.g. workflow portability, family-family triggers, etc).

Is this not a dummy task, however? (The example in the docs PR for family-family trigger is called a dummy task)

Reply:
> Is this not a dummy task, however? (The example in the docs PR for family-family trigger is called a dummy task)

I have accidentally overloaded a term: I (and possibly others) have been calling a non-functional task added for the purposes of simplifying a graph a _dummy_ task. Such a task would be run in _any_ non-live mode (or in live mode with `script = true`). The attraction of running such a task in skip mode is that it will not touch the file system or go through most of the standard job submission pathway.

I will re-examine my docs PR to ensure that this is clear.

Reply:
Unresolving pending response to @MetRonnie 's question.